packages/testing-utils/src/index.js

```diff
@@ -20,15 +20,18 @@ export const cloneableGenerator = generatorFunc => (...args) => {
 
 export function createMockTask() {
   let _isRunning = true
+  let _isAborted = false
   let _result
   let _error
 
   return {
     [TASK]: true,
     isRunning: () => _isRunning,
+    isAborted: () => _isAborted,
```

Comment:
This always returns a constant value - `false`. Shouldn't we expose a similar setter as we do for other "statuses" (`setRunning`, `setError`, etc)? I also wonder - some of those can't be true at the same time, so I think that it would make sense to create an internal `status` variable, initialize it to `running'` and then switch it appropriately based on those setters. That way we won't end up with an impossible task with `t.isRunning() === true && t.isAborted() === true`

Reply:
In general, it's fairly easy to inspect how the real task behaves here:
https://github.com/redux-saga/redux-saga/blob/4da7bd91d1548218e600c43592fc14ec83819d5e/packages/core/src/internal/newTask.js#L118-L141